tests/query/test_project_extension.py

```diff
@@ -99,6 +119,7 @@ def test_with_turbo(self):
         )
 
         assert self.query.get_conditions() == [("project_id", "IN", [2])]
+        assert self.query.get_condition_from_ast() == build_in([2])
```

Comment:
I think it'd be easier to read this if `build_in` also took the left hand side column name, e.g. `build_in("project_id", [2])`, mostly just because it'd be more obviously similar to the previous line.